src/App.js

```diff
@@ -163,7 +88,7 @@ class App extends Component {
             ...this.state.subpages,
             {
                 id: uuidv1(),
-                type: this.state.orderTypes[0].value,
+                type: Object.keys(constantsOrderTypes.orderTypes)[0],
```

Comment:
I think it would be better to just have `DEFAULT_ORDER_TYPE` constant and use it here
```suggestion
                type: DEFAULT_ORDER_TYPE,
```

And this weird construct would be used only once, in `src/shared/constants/constants.orderTypes.js`
`export const DEFAULT_ORDER_TYPE = Object.keys(orderTypes)[0];`